CMake/opengl_config.cmake

```diff
@@ -1,2 +1,6 @@
+if (POLICY CMP0072)
```

Comment:
Note that this is cherry-picked from upstream so that vlgnd is used by default: https://github.com/IntelRealSense/librealsense/blob/master/CMake/opengl_config.cmake#L1